cluster/ubuntu/util.sh

```diff
@@ -358,13 +358,13 @@ function provision-master() {
                             create-kube-apiserver-opts "${SERVICE_CLUSTER_IP_RANGE}" "${ADMISSION_CONTROL}" "${SERVICE_NODE_PORT_RANGE}"; \
                             create-kube-controller-manager-opts "${NODE_IPS}"; \
                             create-kube-scheduler-opts; \
-                            create-flanneld-opts "127.0.0.1"; \
                             sudo -p '[sudo] password to start master: ' cp ~/kube/default/* /etc/default/ && sudo cp ~/kube/init_conf/* /etc/init/ && sudo cp ~/kube/init_scripts/* /etc/init.d/ ;\
                             sudo groupadd -f -r kube-cert; \
                             ${PROXY_SETTING} sudo -E ~/kube/make-ca-cert.sh ${MASTER_IP} IP:${MASTER_IP},IP:${SERVICE_CLUSTER_IP_RANGE%.*}.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local; \
                             sudo mkdir -p /opt/bin/ && sudo cp ~/kube/master/* /opt/bin/; \
                             sudo service etcd start; \
-                            sudo FLANNEL_NET=${FLANNEL_NET} -b ~/kube/reconfDocker.sh "a";"
+                            sudo FLANNEL_NET=${FLANNEL_NET} ~/kube/reconfDocker.sh "a"; \
+                            create-flanneld-opts "127.0.0.1";"
```

Comment:
We have tested it in clean Ubuntu 14.04.3 LTS deployments, with 3 nodes ( ai, i, i ).
Configuring flannel over a B class network (172.16.0.0/16).
At the end of the installation flanneld and docker are up.
Flannel interface and docker interface might not be linked to the same subnet because /run/flannel/subnet.env does not exist  when source /run/flannel/subnet.env is executed at reconfDocker.sh
Thus, we can not reach a pod from any other pod located in a different node.